src/Process.cxx

```diff
@@ -196,6 +210,10 @@ namespace ldmx {
 
             for (auto module : sequence_) module->onFileClose(outFile);
             
+            runHeader.setRunEnd(std::time(nullptr));
+            std::stringstream ss;
+            ss << runHeader;
+            ldmx_log(info) << ss.str();
```

Comment:
Do you really want to print out the run header at the end of the run? Maybe just make the log level something other than info. 

I'm thinking of the main use case where we are running on the batch and we don't usually see any of this info anyways.

Reply:
That's exactly why I think printing it to the `info` channel is the best mthod. In batch, the log level will be higher than info so it won't be printed.

Reply:
Oh I see, printing the RunHeader to the stringstream always happens... I will look into ways around this.